cmake/kokkos_arch.cmake

```diff
@@ -378,6 +378,17 @@ IF (KOKKOS_ARCH_HSW)
   )
 ENDIF()
 
+IF (KOKKOS_ARCH_RISC64_rv64imafdcv)
+  COMPILER_SPECIFIC_FLAGS(
+    COMPILER_ID KOKKOS_CXX_HOST_COMPILER_ID
+        Cray    NO-VALUE-SPECIFIED
+        Intel   NO-VALUE-SPECIFIED
+        MSVC    NO-VALUE-SPECIFIED
+        NVHPC   NO-VALUE-SPECIFIED
+        DEFAULT -march=rv64imafdcv 
```

Comment:
You require to use whitespaces instead of tabs.
See https://cloud.cees.ornl.gov/jenkins-ci/job/Kokkos/job/PR-6771/1/pipeline-console/log?nodeId=25